apr/tests/test.bats

```diff
@@ -0,0 +1,15 @@
+@test "package directory for package ident ${TEST_PKG_IDENT} exists" {
```

Comment:
I don't know if this test is required. We're not supposed to test the functionality of hab, just the package we're installing  with hab. If there was some problem creating the directory, this would be picked up ion following tests, too.

Reply:
True, this is a test of the plan installation and not a test of the plan.
I used a test like this in the [7zip example in the test standard document](https://github.com/habitat-sh/core-plans/blob/master/docs/dev/policy_documents/standards-for-tests.md#linuxbats-tests) and the intention was to give a clear failure if the package isn't installed at the time the tests run.

I think it's OK to keep this test but we should consider removing the test from the 7zip example in the test standard - I'll do a PR for this later.